### PR TITLE
Compatible with python3

### PR DESCRIPTION
**Pull request by chrisperez626**

### CODE REVIEW

File: Classes/Square.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
-import constants
```
Comment (aylusltd) [inline Classes/Square.py:1]: why did you switch import methods?

**Review by aylusltd — COMMENTED** [review]
What's with the change in every file to `import * from constants`? This shouldn't be in this PR unless it's necessary for Python3 migration.

**Comment by chrisperez626** [comment]
It would not run unless i changed it. I tried leaving it and constants wasn't coming up as an object for some reason but every point where it called import constants it wasn't working. Hence why it was changed everywhere. Also that shouldn't be like that in Square.py it should be how i put every other import of the file.

**Comment by aylusltd** [comment]
fixed and merged. It's because you imported * from tkinter which contained an object called constants.

**Comment by chrisperez626** [comment]
gotcha that makes sense there was an error at some point saying tkinter.constants. I should've picked that up.

**Pull request merged**